torch/csrc/inductor/aoti_torch/shim_common.cpp

```diff
@@ -147,6 +147,13 @@ AOTITorchError aoti_torch_empty_strided(
   });
 }
 
+AOTITorchError aoti_torch_new_tensor(AtenTensorHandle* ret) {
+  AOTI_TORCH_CONVERT_EXCEPTION_TO_ERROR_CODE({
+    at::Tensor* out_tensor = new at::Tensor();
```

Comment:
Who is responsible for deallocating this?

Reply:
In the codege, there will be a `create_raii_tensor_handle_for_temp` call latter, which binds a deleter for this.